tests/test-http-client-transport/src/test/java/org/eclipse/jetty/http/client/TransportProvider.java

```diff
@@ -35,21 +35,16 @@
         String transports = System.getProperty(Transport.class.getName());
 
         if (!StringUtil.isBlank(transports))
-        {
-            return Arrays.stream(transports.split("\\s*,\\s*"))
-                    .map(Transport::valueOf);
-        }
+            return Arrays.stream(transports.split("\\s*,\\s*")).map(Transport::valueOf);
 
-        // TODO #2014 too many test failures, don't test unix socket client for now.
-        // if (OS.IS_UNIX)
-        //     return Transport.values();
+         if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("linux"))
```

Comment:
There's an easier way to do this for test cases.

See example usage in https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-start/src/test/java/org/eclipse/jetty/start/PathMatchersSearchRootTest.java

Reply:
Thanks, done.

Reply:
Am I missing a commit?  didn't see this changed.